src/screens/Customers/CustomerMain.tsx

```diff
@@ -1,29 +1,124 @@
 import React, { useEffect, useState } from 'react';
-import { CustomerRecord } from '../../lib/airtable/interface';
-import { getAllCustomers } from '../../lib/airtable/request';
+import { CustomerRecord, SiteRecord } from '../../lib/airtable/interface';
 import { Link, RouteComponentProps } from 'react-router-dom';
+import { store } from '../../lib/redux/store';
+import { createStyles, FormControl, FormHelperText, MenuItem, Select, Theme, ThemeProvider, withStyles } from '@material-ui/core';
+import UserSearchBar from '../../components/UserSearchBar';
+import CustomerCard from '../../components/CustomerCard';
+import BaseScrollView from '../../components/BaseComponents/BaseScrollView';
+import TrieTree from '../../lib/utils/TrieTree';
+import BaseScreen from '../../components/BaseComponents/BaseScreen';
 
-function CustomerMain(props: RouteComponentProps) {
+const styles = (theme: Theme) =>
+  createStyles({
+    title: {
+      width: '153.9px',
+      height: '33.83px',
```

Comment:
This causes awkward spacing between the search bar and title. See screenshots.

With:
![image](https://user-images.githubusercontent.com/35501399/104109555-fdf7c900-5283-11eb-9f3f-a391c0cc0269.png)

Without:
![image](https://user-images.githubusercontent.com/35501399/104109566-1667e380-5284-11eb-835e-1971db0fbf22.png)